app/src/main/java/com/keylesspalace/tusky/entity/EmojiCompatFont.java

```diff
@@ -0,0 +1,321 @@
+package com.keylesspalace.tusky.entity;
+
+import android.content.Context;
+import android.os.AsyncTask;
+import android.util.Log;
+
+import com.google.gson.Gson;
+import com.keylesspalace.tusky.FileEmojiCompatConfig;
+import com.keylesspalace.tusky.R;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import okio.BufferedSink;
+import okio.Okio;
+
+
+// TODO: It might be possible that this is not a class that belongs to the entity-package...
```

Comment:
put it into the util package for now until we figure out a better package structure